### PR TITLE
crash on first view to company screen (bug)

### PR DESCRIPTION
**Pull request by OH296**
- fixes issue #180 and associated issues

### CODE REVIEW

**Comment by renatosilvatimebel** [comment]
yup, seems to have done the trick, thanks @OH296 @GalacticChimp 

image

File: objects/obj_controller/Step_0.gml [hunk 1]
```diff
@@ -724,7 +724,11 @@ if (menu==1) and (managing>0){
                     if (floor(b1/2)==(b1/2)) then b2=(10.43*ma_bio[sel]);
                 }
                 if (ma_bio[sel]>3) then b2=(10.43*ma_bio[sel])-5;
-                temp[114]=string(unit.bionics())+" ("+string(round(max(0,b2)))+"%)";
+                if (unit.bionics()>0){
+                    temp[114]=string("{0}({1}%)",unit.bionics(),round(max(0,b2)));
```
Comment (GalacticChimp) [inline objects/obj_controller/Step_0.gml:728]: I recommend string interpolation, will be much clearer

```suggestion
                    temp[114] = $"{unit.bionics()}({round(max(0,b2))}%)";
```

File: scripts/scr_initialize_custom/scr_initialize_custom.gml [hunk 2]
```diff
@@ -2350,7 +2358,7 @@ function scr_initialize_custom() {
 	scr_restart_variables(1);
 
 
-	var eqi;eqi=0;
+	var eqi=0;
```
Comment (GalacticChimp) [inline scripts/scr_initialize_custom/scr_initialize_custom.gml:2361]: 👍

**Review by GalacticChimp — CHANGES_REQUESTED** [review]
Looks fine, just please resolve my 1 comment and I'll merge

**Review by GalacticChimp — APPROVED** [review]

**Pull request merged**